google-cloud-core/src/main/java/com/google/cloud/HttpTransportOptions.java

```diff
@@ -154,10 +155,26 @@ public void initialize(HttpRequest httpRequest) throws IOException {
         if (readTimeout >= 0) {
           httpRequest.setReadTimeout(readTimeout);
         }
+
+        HttpHeaders headers = httpRequest.getHeaders();
+        headers.set("x-goog-api-client", getXGoogApiClientHeader());
       }
     };
   }
 
+  String getXGoogApiClientHeader() {
+    return String.format(
+        "gl-java/%s %s/%s",
```

Comment:
Presumably, that what `Google-HTTP-Java-Client` tells us in the `user-agent` header, correct? My understanding is that is says the legacy client library for Java was involved in the API call.

Part of the reason I'm not in a rush to add `gdcl` to `x-goog-api-client` is that we have a long history of legacy client library versions (i.e. not GAPIC) out there that don't use `x-goog-api-client`. We need to keep tracking them, so `gdcl` is not super valuable in the short term.